app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def file_from_input():
+    name = input("Enter name of the file: ")
+    is_runing = True
+    while is_runing:
+        content = input("Enter new line of content: ")
+        if content == "stop":
+            is_runing = False
```

Comment:
It would be better to delete line 3, change line 4 to `while True:` and line 7 - `break`

Reply:
ok